src/main/java/de/tum/in/www1/artemis/domain/metis/conversation/Channel.java

```diff
@@ -114,4 +116,9 @@ public Boolean getIsAnnouncementChannel() {
     public void setIsAnnouncementChannel(Boolean announcementChannel) {
         isAnnouncementChannel = announcementChannel;
     }
+
+    @Override
+    public String getHumanReadableNameForReceiver(User sender) {
+        return name;
```

Comment:
You use the getter in all other overwrites of this method

Reply:
Thank you for the suggestion. I replaced it by getName()